scripts/getAdditionalFactoryCollectionItemsNFTCatalog.cdc

```diff
@@ -96,16 +132,25 @@ pub fun fetchNFTCatalog(user: String, collectionIDs: {String : [UInt64]}) : {Str
 			if project != nft!.collectionDisplay!.name {
 			 subCollection = nft!.collectionDisplay!.name
 			}
+
+			var name = nft!.display!.name 
+			if name == "" {
+				name = projectName
+			}
+
+			if nft.editions != nil && nft.editions!.infoList.length > 0 {
```

Comment:
I think in this case we need to iterate and check if there is an edition with no name or «set»